runtime-modules/service-discovery/src/mock.rs

```diff
@@ -11,10 +11,10 @@ pub use sr_primitives::{
 
 use srml_support::{impl_outer_event, impl_outer_origin, parameter_types};
 
-mod bureaucracy_mod {
-    pub use bureaucracy::Event;
-    pub use bureaucracy::Instance2;
-    pub use bureaucracy::Trait;
+mod working_group_mod {
+    pub use working_group::Event;
+    pub use working_group::Instance2;
```

Comment:
might make sense to alias `Instance2` to `StorageWorkingGroup` with `as` to make it easier to understand which instance we are working with? In other places as well if it makes sense, this would be important in future as we add more instances. Also why did we skip Instance1 ?

Reply:
Introduced the alias for instances. Instance1 is reserved for the Forum.